src/main/java/smartthings/cassandra/CassandraLock.java

```diff
@@ -31,22 +33,35 @@ public CassandraLock(CassandraConnection cassandraConnection, int ttl) {
 
 		setupTables();
 
-		insertLock = session.prepare("INSERT INTO databasechangelock(id, lockedby) VALUES (:lockId, :owner) IF NOT EXISTS USING TTL :ttl");
-		insertLock.setConsistencyLevel(ConsistencyLevel.QUORUM);
-
-		deleteLock = session.prepare("DELETE FROM databasechangelock WHERE id = :lockId IF lockedby = :owner");
-		deleteLock.setConsistencyLevel(ConsistencyLevel.QUORUM);
-
-		selectLock = session.prepare("SELECT lockedby, TTL(lockedby) AS ttl FROM databasechangelock WHERE id = :lockId");
-		selectLock.setConsistencyLevel(ConsistencyLevel.SERIAL);
-
-		updateLock = session.prepare("UPDATE databasechangelock USING TTL :ttl SET lockedby = :owner WHERE id = :lockId IF lockedby = :owner");
-		deleteLock.setConsistencyLevel(ConsistencyLevel.QUORUM);
+		insertLock = session.prepare(
+			SimpleStatement.newInstance("INSERT INTO databasechangelock(id, lockedby) VALUES (:lockId, :owner) IF NOT EXISTS USING TTL :ttl")
+				.setConsistencyLevel(DefaultConsistencyLevel.QUORUM)
+		);
+
+		deleteLock = session.prepare(
+			SimpleStatement.newInstance("DELETE FROM databasechangelock WHERE id = :lockId IF lockedby = :owner")
+				.setConsistencyLevel(DefaultConsistencyLevel.QUORUM)
+		);
+
+		selectLock = session.prepare(
+			SimpleStatement.newInstance("SELECT lockedby, TTL(lockedby) AS ttl FROM databasechangelock WHERE id = :lockId")
+				.setConsistencyLevel(DefaultConsistencyLevel.SERIAL)
+		);
+
+		updateLock = session.prepare(
+			SimpleStatement.newInstance("UPDATE databasechangelock USING TTL :ttl SET lockedby = :owner WHERE id = :lockId IF lockedby = :ifowner")
+			.setConsistencyLevel(DefaultConsistencyLevel.QUORUM)
+		);
 	}
 
 	public boolean tryLock() {
-		ResultSet rs = session.execute(insertLock.bind().setInt("lockId", lockId)
-				.setInt("ttl", ttl).setString("owner", owner));
+		ResultSet rs = session.execute(
+			insertLock.boundStatementBuilder()
```

Comment:
Switched `bind` to `boundStatementBuilder` to reduce cost of data copy due to immutable internal implementation.